icons/raspberrypi/raspberrypi-original-wordmark.svg

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   version="1.1"
+   id="svg2"
+   xml:space="preserve"
+   width="3080.5601"
+   height="825.76001"
+   viewBox="0 0 3080.5601 825.76001"
```

Comment:
Please have a look at our guidelines regarding [svg requirements](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#svg-standards). The when you update this pull request, those requirements should be checked by out bot.. Let's see :)